pkg/cli/secretsencrypt/secrets_encrypt.go

```diff
@@ -0,0 +1,212 @@
+		fp := filepath.Join(controlConfig.DataDir, "token")
+		tokenByte, err := ioutil.ReadFile(fp)
+		if err != nil {
+			return controlConfig, err
+		}
+		controlConfig.Token = string(bytes.TrimRight(tokenByte, "\n"))
+	} else {
+		controlConfig.Token = cmds.ServerConfig.Token
+	}
+	controlConfig.EncryptForce = cfg.EncryptForce
+
+	return controlConfig, nil
+}
+
+func Run(app *cli.Context) error {
+	fmt.Println("This command does nothing, use the subcommands")
+	return nil
+}
+
+func Enable(app *cli.Context) error {
+	var err error
+	if err = cmds.InitLogging(); err != nil {
+		return err
+	}
+	controlConfig, err := commandPrep(app, &cmds.ServerConfig)
+	if err != nil {
+		return err
+	}
+	info, err := clientaccess.ParseAndValidateTokenForUser(cmds.ServerConfig.ServerURL, controlConfig.Token, "node")
+	if err != nil {
+		return err
+	}
+	b, err := json.Marshal(server.EncryptionRequest{Toggle: true})
+	if err != nil {
+		return err
+	}
+	if err = info.Put("/v1-"+version.Program+"/encrypt/toggle", b); err != nil {
+		return err
+	}
+	fmt.Println("secrets-encryption enabled")
+	return nil
+}
+
+func Disable(app *cli.Context) error {
+	var err error
+	if err = cmds.InitLogging(); err != nil {
+		return err
+	}
+	controlConfig, err := commandPrep(app, &cmds.ServerConfig)
+	if err != nil {
+		return err
+	}
+	info, err := clientaccess.ParseAndValidateTokenForUser(cmds.ServerConfig.ServerURL, controlConfig.Token, "node")
+	if err != nil {
+		return err
+	}
+	b, err := json.Marshal(server.EncryptionRequest{Toggle: false})
+	if err != nil {
+		return err
+	}
+	if err = info.Put("/v1-"+version.Program+"/encrypt/toggle", b); err != nil {
+		return err
+	}
+	fmt.Println("secrets-encryption disabled")
+	return nil
+}
+
+func Status(app *cli.Context) error {
+	if err := cmds.InitLogging(); err != nil {
+		return err
+	}
+	controlConfig, err := commandPrep(app, &cmds.ServerConfig)
+	if err != nil {
+		return err
+	}
+	info, err := clientaccess.ParseAndValidateTokenForUser(cmds.ServerConfig.ServerURL, controlConfig.Token, "node")
+	if err != nil {
+		return err
+	}
+	data, err := info.Get("/v1-" + version.Program + "/encrypt/status")
+	if err != nil {
+		return err
+	}
+	fmt.Print(string(data))
+	return nil
+}
+
+func Prepare(app *cli.Context) error {
+	var err error
+	if err = cmds.InitLogging(); err != nil {
+		return err
+	}
+	controlConfig, err := commandPrep(app, &cmds.ServerConfig)
+	if err != nil {
+		return err
+	}
+	info, err := clientaccess.ParseAndValidateTokenForUser(cmds.ServerConfig.ServerURL, controlConfig.Token, "node")
+	if err != nil {
+		return err
+	}
+	b, err := json.Marshal(server.EncryptionRequest{
+		Stage: server.EncryptionPrepare,
+		Force: controlConfig.EncryptForce,
+	})
+	if err != nil {
+		return err
+	}
+	if err = info.Put("/v1-"+version.Program+"/encrypt/stage", b); err != nil {
+		return err
+	}
+	fmt.Println("prepare completed successfully")
+	return nil
+}
+
+func Rotate(app *cli.Context) error {
+	var err error
```

Comment:
Doesn't look like this needs to be declared here or in the function below.

Reply:
Reduced the boilerplate more, I still want to keep all the individual functions.